core/src/main/java/org/apache/brooklyn/core/entity/AbstractEntity.java

```diff
@@ -1737,7 +1738,6 @@ public boolean removeAllPolicies() {
      * {@link BasicEnricherSupport}.
      */
     @Beta
-    // TODO revert to private when config() is reverted to return SensorSupportInternal
```

Comment:
I believe this still needs to be addressed, though the comment has copy&paste error referring to sensors.